docs/modules/ROOT/pages/how-to-guides/proc_release_plan.adoc

```diff
@@ -0,0 +1,60 @@
+= Creating a `ReleasePlan` object
+
+The development team creates a `ReleasePlan` object in the developer workspace. The `ReleasePlan` object includes a reference to the application that the development team wants to release, along with the namespace and workspace where the application is suppose to release.
+
+.*Prerequisites*
+
+* You have an existing Development workspace.
+* Ensure you have installed `oc`.
+* You have completed the steps listed in the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/[Getting started in the CLI] page.
+
+.*Procedures*
+
+. Create a `ReleasePlan.yaml` object locally.
+
++
+*Example `ReleasePlan.yaml` object*
+
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan <.>
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true'
+ name: sre-production <.>
+ namespace: dev-workspace <.>
+spec:
+ application: demo-app <.>
+ data: <key> <.>
+ pipelineRef: <pipeline_ref> <.>
+ serviceAccount:  <service-account> <.>
+ target: managed-workspace <.>
+----
+
++
+<.> The name of the resource that you are creating.
+<.> The name of the release plan.
+<.> The development team's workspace.
+<.> The name of the application that you want to deploy to the managed workspace.
+<.> An unstructured key used for providing data for the release Pipeline.
```

Comment:
This is optional

Reply:
@johnbieren So when you say this is optional, is this for lines # 39 through #41 or just #41? Sorry, I did not get this.

Reply:
This (line 40) is the only one that is optional that doesn't say it. In the spec, application and target are required. data, pipelineRef and serviceAccount are optional. It seems that pipelineRef and serviceAccount have `Optional:` in their current descriptions, but `data` does not